x-pack/plugins/rollup/public/crud_app/sections/job_list/job_table/job_table.js

```diff
@@ -0,0 +1,255 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+
+import {
+  EuiFieldSearch,
+  EuiPage,
+  EuiSpacer,
+  EuiTable,
+  EuiTableBody,
+  EuiTableHeader,
+  EuiTableHeaderCell,
+  EuiTablePagination,
+  EuiTableRow,
+  EuiTableRowCell,
+  EuiTitle,
+  EuiPageBody,
+  EuiPageContent,
+  EuiToolTip,
+  EuiLink,
+} from '@elastic/eui';
+
+import { JobStatus } from '../job_status';
+
+const COLUMNS = [{
+  name: 'ID',
+  fieldName: 'id',
+}, {
+  name: 'Status',
+  fieldName: 'status',
+  render: ({ status, rollupCron }) => {
+    return (
+      <EuiToolTip placement="top" content={`Cron: ${rollupCron}`}>
+        <JobStatus status={status} />
+      </EuiToolTip>
+    );
+  },
+}, {
+  name: 'Index pattern',
+  truncateText: true,
+  fieldName: 'indexPattern',
+}, {
+  name: 'Rollup index',
+  truncateText: true,
+  fieldName: 'rollupIndex',
+}, {
+  name: 'Delay',
+  fieldName: 'rollupDelay',
```

Comment:
this could use a custom render to show `None` for no value like in the flyout:
`render(rollupDelay) => <Fragment>{rollupDelay || 'None'}</Fragment>`

Reply:
Done